src/components/QuestionnaireForm/QuestionnaireForm.jsx

```diff
@@ -85,9 +87,16 @@ export default class QuestionnaireForm extends Component {
     this.loadAndMergeForms(this.state.savedResponse);
   }
 
+  getRetrieveSaveQuestionnaireUrl = () => {
+    // read configuration 
+    let updateDate = new Date();
+    updateDate.setDate(updateDate.getDate() - ConfigData.QUESTIONNAIRE_EXPIRATION_DAYS);
+    return "QuestionnaireResponse?" + "_updateDate=" + updateDate.toISOString().split('T')[0];
```

Comment:
```suggestion
    return "QuestionnaireResponse?" + "_lastUpdated=gt" + updateDate.toISOString().split('T')[0];
```

Reply:
It should be fixed now. Sorry about the back and forth :-)